test/decompiler/reference/engine/target/surface-h_REF.gc

```diff
@@ -1096,7 +1096,7 @@
    (-> v1-41 mult-hook)
    (the-as (function surface surface surface int none) nothing)
    )
-  (set! (-> v1-41 touch-hook) nothing)
+  (set! (-> v1-41 touch-hook) (the-as (function symbol) nothing))
```

Comment:
`nothing` definitely doesn't return anything, I think `touch-hook` should be a `(function none)`